src/components/RandomDog.js

```diff
@@ -1,15 +1,29 @@
 import React from "react";
 import "./RandomDog.css";
 
-export default function RandomDog() {
+function RandomDog(props) {
+  
+  // {
+  //     "message": "https://images.dog.ceo/breeds/hound-ibizan/n02091244_4281.jpg",
+  //     "status": "success"
+  // }
+
+console.log("props",props)
+
   return (
     <div className="RandomDog">
       <h2 className="RandomDog-title">Random Dogs</h2>
-      <img className="RandomDog-image" src="http://via.placeholder.com/300x300" />
+      <img className="RandomDog-image" src={props.randomDogImage} />
       <p>
-        <button className="RandomDog-button">Save Image</button>
-        <button className="RandomDog-button">Next Dog</button>
+        <button onClick={props.savePhotos} className="RandomDog-button">
```

Comment:
I think this onClick is wrong, savePhotos is your array for saving your photo, please check and change it